hrm-service/docker-entrypoint

```diff
@@ -4,6 +4,9 @@
 node ./db-migrate
 
 # Execute resources-service migrations (move once the service lives in it's own container)
-node ./node_modules/@tech-matters/resources-service/db-migrate
+cd ./resources-service
+node ./create-db
```

Comment:
TBH the only oddity I can think of for misconfigured env vars would be creating a second resources user, and I can't think of a good reason why the resource's user name is controlled by a var when the hrm one is hardcoded - @GPaoloni was there a reason for this I'm missing?

But that doesn't address the security concern. I'm curious though, in the migrations version the user running the HRM migrations needed the privileges to create the resources user, which is the same privileges the create-db script user needs, so I'm not sure if there's any difference from that angle?

That said, I'm ok with it being a manual step, given it's easy to do and not needed often but we will need to document it for when we set up new HRM backend environments in the future

Reply:
> But that doesn't address the security concern. I'm curious though, in the migrations version the user running the HRM migrations needed the privileges to create the resources user, which is the same privileges the create-db script user needs, so I'm not sure if there's any difference from that angle?

There isn't any difference, I just hadn't raised the concern yet. ;)

Reply:
RDS instances and DB permissions are one of the few places where, until they are so frequently being created/modified that it becomes cumbersome, I tend to minimize automation. The risk/reward ratio is just so much different than pretty much any other part of an infrastructure that I tend to handle them with kid gloves until there is a VERY good reason not to.

They are the one exception to the livestock not pets philosophy in most of the systems that I build. Because they are almost ALWAYS pets, at lest the production versions are.

There are cases where things like dynamic dev environments in the cloud mean automating creation of RDS instances automatically... but even then I tend to treat production and staging differently.